src/templates/luneClient.hbs

```diff
@@ -32,6 +32,7 @@ function applyMixins(derivedCtor: any, constructors: any[]) {
     })
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unsafe-declaration-merging
```

Comment:
Interesting, what are we actually merging here?

Reply:
There's an interface called LuneClient and a class called LuneClient in the same file.

Reply:
Yeah, but the class is declared before the interface so at this point nothing's being merged yet?

Reply:
`eslint` see it.